src/transactions/test/TxEnvelopeTests.cpp

```diff
@@ -710,6 +710,52 @@ TEST_CASE("txenvelope", "[tx][envelope]")
                         });
                     }
 
+                    SECTION("signature verification fails due to too many "
```

Comment:
Can you give this section a name that reflects the fact that what you are really demonstrating is that multiple accounts can be modified even on transaction failure? If someone is looking for this, they won't find it with the current name.